app/helpers/host_helper/textual_summary.rb

```diff
@@ -92,36 +92,48 @@ def textual_generate_openstack_status
       all_count           = x.system_services.count
       configuration_count = x.filesystems.count
 
+      running_link = url_for_only_path(:controller         => controller.controller_name,
+                                       :action             => 'host_services',
+                                       :id                 => @record,
+                                       :db                 => controller.controller_name,
+                                       :host_service_group => x.id,
+                                       :status             => :running)
```

Comment:
`running_link `, `failed_link ` and  `all_link ` have only one difference in `:status`. I think it's worth moving it in a method (maybe `configuration_link` as well). What do you think?

Reply:
Yep, fixed.